code/modules/projectiles/guns/projectile/automatic/blackguard.dm

```diff
@@ -12,14 +12,15 @@
 	origin_tech = list(TECH_COMBAT = 7, TECH_MATERIAL = 2)
 	slot_flags = SLOT_BACK
 	load_method = SINGLE_CASING|MAGAZINE
-	mag_well = MAG_WELL_HRIFLE|MAG_WELL_DRUM
+	mag_well = MAG_WELL_HRIFLE //Haha, no
```

Comment:
Add back the drum and read the desk, this gun was made to take the drum

Reply:
Agree with Trilby. Unnecessary change for a weapon that is one of the few able to take these drums and that was designed with this in mind

Reply:
Reverted but not the brace penalty, if long arm has one this should too